cmd/ddev/cmd/scripts/test_ddev.sh

```diff
@@ -189,19 +187,23 @@ DDEV_DEBUG=true ddev start -y || ( \
   printf "Start failed.\n" && \
   exit 1 )
 
+host_http_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.services.web.host_http_url' 2>/dev/null)
 http_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.httpURLs[0]' 2>/dev/null)
 https_url=$(ddev describe -j | docker run -i --rm ddev/ddev-utilities jq -r  '.raw.httpsURLs[0]' 2>/dev/null)
 
 header "Curl of site from inside container"
 ddev exec curl --fail -I http://127.0.0.1
 
-header "curl -I of ${http_url} from outside"
+header "curl -I of ${host_http_url} (web container http docker bind port) from outside"
+curl --fail -I ${host_http_url}
+
+header "curl -I of ${http_url} (router http URL) from outside"
 curl --fail -I "${http_url}"
 
-header "Full curl of ${http_url} from outside"
+header "Full curl of ${http_url} (router https URL) from outside"
```

Comment:
Please review these curls, because it says `router https URL` but runs it for `${http_url}`

Reply:
Thanks, fixed